test/tests/functional/pbs_node_buckets.py

```diff
@@ -452,6 +452,51 @@ def test_not_buckets(self):
                                   sharednode=False, vnodes_per_host=4)
         self.check_normal_path(sel='2:ncpus=8')
 
+    @timeout(450)
+    def test_multi_vnode_resv(self):
```

Comment:
Can you please add some comments in the test? It wasn't obvious what it was doing.